packages/react/src/hooks/useDataChannel.ts

```diff
@@ -23,25 +24,24 @@ type UseDataChannelReturnType<T extends string | undefined = undefined> = {
  * // Send messages to all participants via the 'chat' topic.
  * const { message: latestMessage, send } = useDataChannel('chat', (msg) => console.log("message received", msg));
  * ```
+ *
+ * @example
+ * ```tsx
+ * // Receive all messages (no topic filtering)
+ * const { message: latestMessage, send } = useDataChannel((msg) => console.log("message received", msg));
+ * ```
+ *
  * @public
  */
 export function useDataChannel<T extends string>(
   topic: T,
   onMessage?: (msg: ReceivedDataMessage<T>) => void,
 ): UseDataChannelReturnType<T>;
+
 /**
- * The `useDataChannel` hook returns the ability to send and receive messages.
- * @remarks
- * There is only one data channel. Passing a `topic` does not open a new data channel.
- * It is only used to filter out messages with no or a different `topic`.
- *
- * @example
- * ```tsx
- * // Send messages to all participants
- * const { message: latestMessage, send } = useDataChannel('chat', (msg) => console.log("message received", msg));
- * ```
- * @public
- */
+ * Overload for `useDataChannel` without a topic. See `useDataChannel<T>` for more usage examples.
+ * @internal
```

Comment:
making it internal seems wrong though, that would suggest to not use this overload

Reply:
yeah i spent a while on this but can't find another way to mark it for doc exclusion, unfortunately

Reply:
maybe @Ocupe has an idea wrt api extractor?

Reply:
I think this should do it on the docs generation part: https://github.com/livekit/components-js/pull/1138

Reply:
perfect - once that's merged i'll adjust this so the one that gets documented will have the docstring describing both usages (unclear which one will be used from that PR, but it doesn't matter either way)

Reply:
the one that's going to be used should always be the first one that's defined

Reply:
ok perfect - i just tested and it seems to work! updated